plan/validate.go

```diff
@@ -52,12 +54,21 @@ func isDockerImage(v *compiler.Value) bool {
 	return plancontext.IsFSValue(v.Lookup("rootfs")) && v.Lookup("config").Kind() == cue.StructKind
 }
 
-func isPlanConcrete(p *compiler.Value, v *compiler.Value) error {
+func isPlanConcrete(p *compiler.Value, v *compiler.Value) (err error) {
```

Comment:
Nit: remove `err`, we are not using that syntax in our codebase.
Is there another way to forward err from the routine to the function's return type?

Reply:
This need for receive panic error.

Reply:
In fact. I think we could remove the 
https://github.com/dagger/dagger/pull/2710/files#diff-a0113f2e52d8cf56b126333062d3901a0ba24847d5d9f76d05f74b3dfea21921R133

Reply:
It's okay :D